src/AMQPEventsReceiver.js

```diff
@@ -20,25 +20,36 @@ class AMQPEventsReceiver extends EventEmitter {
   /**
    * @constructor
    * @param {amqplib.Connection} amqpConnection
-   * @param {String} queueName
+   * @param {Object} params
+   * @param {String} [params.queueName=''] queue for feedback from AMQPRPCServer,
+   *    default is '' which means auto-generated queue name, should correspond with AMQPEventsSender
    */
-  constructor(amqpConnection, queueName = '') {
+  constructor(amqpConnection, params = {}) {
     super();
+
     this._connection = amqpConnection;
-    this._queueName = queueName;
+
+    this._params = params;
+    params.queueName = params.queueName || '';
+    params.TTL = params.TTL || AMQPEventsReceiver.TTL;
+    this._queueName = params.queueName;
+
+    this._channel = null;
   }
 
   /**
    * Begin to listen for messages from amqp
    * @returns {Promise<String>} name of endpoint to send messages
    */
-  async receive() {
-    assert(!this._channel && !this._queue, 'Already receiving');
+  async start() {
+    assert(!this._channel, 'Already started');
     this._channel = await this._connection.createChannel();
-    const queue = await this._channel.assertQueue(this._queueName, {
-      durable: true
-    });
-    this._queueName = queue.queue;
+    if (this._queueName === '') {
+      const queue = await this._channel.assertQueue(this._queueName, {
+        durable: true,
```

Comment:
anonymous queue, i think it should not be durable, as if client dies, none ever will be able to consume from it.
I think, it should be exclusive, so it should be autodestroyed (and sequentially, we don't need to delete it on disconnect)